### PR TITLE
added support for jdk 16, migrated to jakarta jaxb, added support for…

### PR DESCRIPTION
**Pull request by toggm**
… additional launch parameters after initialization

### CODE REVIEW

File: src/main/java/fxlauncher/Launcher.java [hunk 1]
```diff
@@ -248,12 +259,12 @@ private void startApplication() throws Exception {
 				appParams.getRaw().addAll(params.getRaw());
 				appParams.getUnnamed().addAll(params.getUnnamed());
 			}
-			PlatformImpl.setApplicationName(app.getClass());
+			//Platform.setApplicationName(app.getClass());
```
Comment (robstoll) [inline src/main/java/fxlauncher/Launcher.java:263]: Intentional change?  if so please remove uncommented code

File: src/main/java/fxlauncher/ParametersWithAdditionalNamedParameters.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    public ParametersWithAdditionalNamedParameters(Application.Parameters delegate, Map<String, String> additionalNamedParameters) {
+        this.delegate = delegate;
+        this.combinedNamedParameters = combineNamedParameters(delegate, additionalNamedParameters);
+    }
+
+    private Map<String, String> combineNamedParameters(Application.Parameters delegate, Map<String, String> additionalNamedParameters) {
+        Map<String, String> combinedNamedParameters = new HashMap<>();
+        combinedNamedParameters.putAll(delegate.getNamed());
+        combinedNamedParameters.putAll(additionalNamedParameters);
```
Comment (robstoll) [inline src/main/java/fxlauncher/ParametersWithAdditionalNamedParameters.java:23]: is it OK that one can override existing parameters with additionalNamedParameters?

**Review by robstoll — CHANGES_REQUESTED** [review]

File: src/main/java/fxlauncher/Launcher.java (hunk 1 above)
Comment (toggm) [reply src/main/java/fxlauncher/Launcher.java:263]: Removed as access to internal PlatformImpl is not allowed anymore. Didn't find a proper other implementation way but kept comment be able to support setting correct application name later

File: src/main/java/fxlauncher/ParametersWithAdditionalNamedParameters.java (hunk 2 above)
Comment (toggm) [reply src/main/java/fxlauncher/ParametersWithAdditionalNamedParameters.java:23]: We even need this to be able to inject uri after starting the application

File: src/main/java/fxlauncher/Launcher.java (hunk 1 above)
Comment (robstoll) [reply src/main/java/fxlauncher/Launcher.java:263]: I suggest you document this accordingly
```suggestion
                      // Removed as access to internal PlatformImpl is not allowed anymore. Didn't find a proper other implementation way but kept comment be able to support setting correct application name later
			//Platform.setApplicationName(app.getClass());
```

File: src/main/java/fxlauncher/ParametersWithAdditionalNamedParameters.java (hunk 2 above)
Comment (robstoll) [reply src/main/java/fxlauncher/ParametersWithAdditionalNamedParameters.java:23]: Good

**Review by robstoll — APPROVED** [review]

**Pull request merged**